packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolygonController.m

```diff
@@ -192,14 +156,4 @@ - (bool)hasPolygonWithIdentifier:(NSString *)identifier {
   return self.polygonIdentifierToController[identifier] != nil;
 }
 
-+ (GMSMutablePath *)getPath:(NSDictionary *)polygon {
-  NSArray *pointArray = polygon[@"points"];
-  NSArray<CLLocation *> *points = [FLTGoogleMapJSONConversions pointsFromLatLongs:pointArray];
-  GMSMutablePath *path = [GMSMutablePath path];
-  for (CLLocation *location in points) {
```

Comment:
This array conversion logic was duplicated in several places, so I consolidated it to the new `FGMGetPathFromPoints`.